ert/exceptions/_exceptions.py

```diff
@@ -41,3 +41,12 @@ def __init__(self, message: str) -> None:
 class ElementMissingError(StorageError):
     def __init__(self, message: str) -> None:
         self.message = message
+
+
+class ExperimentError(ErtError):
+    def __init__(
+        self, message: str, experiment_name: Optional[str], suggest_clean: bool = False
+    ) -> None:
+        self.message = message
+        self.experiment_name = experiment_name
+        self.suggest_clean = suggest_clean
```

Comment:
Is this always going to be true?

Either way, maybe rename it to something indicating what likely caused the error, and not what the outer scope should do? /nitpick

Reply:
You are right (on both counts). The idea was to bring some intelligence/flexibility on how the cli could choose to handle and report certain errors. But to be honest that seems overkill at these point. I will simplify this.